cmd/auth/login.go

```diff
@@ -56,6 +60,38 @@ var loginCmd = &cobra.Command{
 			return err
 		}
 
+		if configureCluster {
+			err := root.MustWorkspaceClient(cmd, args)
+			if err != nil {
+				return err
+			}
+			ctx := cmd.Context()
+			w := root.WorkspaceClient(ctx)
+
+			promptSpinner := cmdio.Spinner(ctx)
+			promptSpinner <- "Loading names for Clusters drop-down."
```

Comment:
This looks copied from the generated code; in the context of this command the title should be updated to e.g. "Loading list of clusters to select from" or something.